### PR TITLE
Release/1.3.0

### PR DESCRIPTION
**Pull request by eymar**

### CODE REVIEW

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatability.kt [hunk 1]
```diff
@@ -7,6 +7,7 @@ internal object ComposeCompilerCompatability {
     private val kotlinToCompiler = sortedMapOf(
         "1.7.10" to "1.3.0",
         "1.7.20" to "1.3.2.2",
+        "1.8.0" to "1.4.0-alpha-01",
```
Comment (sschuberth) [inline gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatability.kt:10]: According to https://developer.android.com/jetpack/androidx/releases/compose-kotlin#pre-release_kotlin_compatibility 1.4.0-alpha-01 has only Kotlin 1.7.20 compatibility. So shouldn't this be 1.4.0-dev-k1.8.0-33c0ad36f83 instead (see [here](https://androidx.dev/storage/compose-compiler/repository))?

Comment (eymar) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatability.kt:10]: 1.4.0-alpha-01 is published by jetbrains.  The versions listed here are not published by Google (Jetpack Compose). 
We have them here: https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/compiler/compiler/ (Note a different artifact id - org.jetbrains.compose.compiler:compiler, not androidx.compose.compiler:compiler)
And stable versions will be published in mavenCentral later

Comment (eymar) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatability.kt:10]: JFYI: Compose Multiplatform 1.3.0-rc02 has been published if you want to give a try

Comment (hfhbd) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatability.kt:10]: @eymar I tried it, but got an error on iOS with compose plugin and serialization plugin: https://github.com/JetBrains/compose-jb/issues/2635

File: compose/buildSrc/src/main/kotlin/ComposePlatforms.kt [hunk 2]
```diff
@@ -25,6 +25,7 @@ enum class ComposePlatforms(vararg val alternativeNames: String) {
     WatchosX64("WatchOs"),
     WatchosSimulatorArm64("WatchOs"),
     LinuxX64("Linux"),
+    LinuxArm64("Linux"),
```
[the remark this answers is not in the document]
Comment (eymar) [reply compose/buildSrc/src/main/kotlin/ComposePlatforms.kt:28]: It's only for compose.runtime :) 
it was missed.

[the remark this answers is not in the document]
Comment (hfhbd) [reply compose/buildSrc/src/main/kotlin/ComposePlatforms.kt:28]: Yes, but that's fine for other use-cases.

**Comment by eymar** [comment]
closing this PR in favor of https://github.com/JetBrains/compose-jb/pull/2651

**Pull request closed without merging**